app/views/incidents/_form.haml

```diff
@@ -8,6 +8,19 @@
       %ul
         - incident.errors.full_messages.each do |message|
           %li= message
+  .field
+    = form.label :latitude
+    = form.text_field :latitude, id: :incident_latitude
+  .field
+    = form.label :longitude
+    = form.text_field :longitude, id: :incident_longitude
+  %i Place a pin on the map below to fill the latitude and longitude.
+  .no-geocode-alert
+    Google could not place the location of this incident on the map:
+    %li= incident.driver_incident_report.full_location include_state: true
```

Comment:
Bullet looks a little odd; it will only ever be a one-item list.

"Google could not place "{location}" on the map. The map will be centered on PVTA"